src/main/kotlin/com/github/seliba/devcordbot/command/context/Context.kt

```diff
@@ -72,14 +72,14 @@ data class Context(
     /**
      * The member of the [author].
      */
-    val member: Member
-        get() = message.member!! //CommandClient ignores webhook messages, so this cannot be null
+    val member: Member?
+        get() = if (message.isFromType(ChannelType.TEXT)) message.member else bot.guild.getMemberById(author.id)
```

Comment:
Would prefer JDA's approach in throwing an exception when type != TEXT, since the dev should know whether he/she/it can use the member or not

Reply:
I need the member for permission checking in private chat